Game/RainMeadow.GameHooks.cs

```diff
@@ -53,11 +53,25 @@ private void GameHooks()
 
             // Arena specific
             On.GameSession.AddPlayer += GameSession_AddPlayer;
-        
+
             IL.Menu.SleepAndDeathScreen.GetDataFromGame += SleepAndDeathScreen_FixNullKarmaLadder;
         }
 
-        private void SleepAndDeathScreen_FixNullKarmaLadder(ILContext il) {
+        private string Options_GetSaveFileName_SavOrExp(On.Options.orig_GetSaveFileName_SavOrExp orig, Options self)
+        {
+            if (self.saveSlot != 0)
+            {
+                return "online_sav" + (self.saveSlot + 1);
+            }
+            return "online_sav";
```

Comment:
Check if we're online before using online saves

```suggestion
            if (OnlineManager.lobby != null) 
            {
                        if (self.saveSlot != 0)
                        {
                            return "online_sav" + (self.saveSlot + 1);
                        }
                        return "online_sav";
            
            }
```

Reply:
left over mb